teuthology/test/task/test_install.py

```diff
@@ -89,9 +90,7 @@ def test_verify_ceph_version_failed(self, m_get_package_version,
         gb.project = "ceph"
         m_gitbuilder_project.return_value = gb
         m_get_package_version.return_value = "0.89.1"
-        config = Mock()
-        # when it looks for config.get('extras') it won't find it
-        config.get.return_value = False
+        config = dict()
```

Comment:
It will fail with `TypeError: argument of type 'Mock' is not iterable` if to check key presence in target method like this:
```
if 'extras' in config
```
instead of
```
if config.get("extras")
```